### DISCUSSION TITLE
Upgrade to latest version

### DISCUSSION THREAD

**Discussion by avishka-codimite** [react-native-tvos/react-native-tvos#378] [Q&A] [answered]
Can I know how to upgrade to the latest version of react-native-tvos? I did these steps to upgrade to the latest version.

Increase the version of react-native in  package.json `"react-native": "npm:react-native-tvos@0.69.6-0"` and then `npm install`

I getting this error after do that 

**TypeError: undefined is not an object (evaluating 'ReactCurrentActQueue$1.isBatchingLegacy')**

**Comment by douglowder** [comment, marked as answer]
I would suggest reverting your manual change to the version, then execute

`npx react-native upgrade`

**Reply by Pnlvfx** [reply]
i'm trying to update from 0.77 to 0.78 using this, does it still exist on the cli?
